pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/PricingManager/Condition/DateRange.php

```diff
@@ -103,10 +103,10 @@ public function fromJSON($string)
     {
         $json = json_decode($string);
 
-        $starting = new \DateTime(strtotime($json->starting));
+        $starting = \DateTime::createFromFormat('Y-m-d\TH:i:s', $json->starting);
         $starting->setTime(0, 0, 0);
 
-        $ending = new \DateTime(strtotime($json->ending));
+        $ending = \DateTime::createFromFormat('Y-m-d\TH:i:s', $json->ending);
```

Comment:
hmm, toJSON() needs to form a value that can be read by fromJSON() ... there we have timestamp vs. datestring

Reply:
the difference in types probably comes from the extsjs datefield that is configured in https://github.com/pimcore/pimcore/blob/master/pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/Resources/public/js/pricing/config/item.js#L659